src/main/groovy/net/foragerr/jmeter/gradle/plugins/JMPluginExtension.groovy

```diff
@@ -17,6 +17,7 @@ class JMPluginExtension {
     Boolean enableExtendedReports = true
 
 	List<File> jmTestFiles = null
+        String srcDir = null
```

Comment:
`File testFileDir` is already the folder that is scanned for .jmx files. That property should be reused for user defined folder instead of creating a new property. 

There should be an edit to the `TaskJMInit`, to check if the user set testFileDir, and if they didn't to assign the default value for the plugin to use. TaskJMRun shouldn't have to change at all to switch directories